wagtailio/blog/models.py

```diff
@@ -56,6 +63,17 @@ def serve(self, request):
     )
 
 
+class BlogPageRelatedPage(Orderable):
+    parent = ParentalKey("blog.BlogPage", related_name="page_related_pages")
```

Comment:
The AC was a bit unclear in that it says "As a user I can see related pages (max two) based on category and publish date.". 
I think this is the right approach.

suggestion: considering this is very specific, let's call it as such. i.e. `related_posts`

Reply:
Thanks, updated in f225dad